packages/frontend/src/view/pages/home/components/HomeStatistics.tsx

```diff
@@ -76,12 +75,25 @@ function StatisticsItem({
   return (
     <div
       className={classNames(
-        'flex flex-col justify-center text-center',
+        'flex flex-col items-center justify-between gap-4 text-center leading-none',
         className
       )}
     >
-      <div className="text-gray-400 text-lg font-medium">{title}</div>
-      <div className="text-bold text-[50px] text-brand">{formatInt(value)}</div>
+      <span className="text-[20px] font-semibold">{title}</span>
+      <span className="text-[50px] font-extrabold text-brand">
+        {formatStatisticsCount(value)}
+      </span>
+      <span className="hidden text-sm font-semibold text-zinc-500 md:inline">
+        # of all current {title.toLowerCase()}
+      </span>
     </div>
   )
 }
+
+function formatStatisticsCount(count: number) {
```

Comment:
We could use the number formatting function we have in L2BEAT, so that we display nicer numbers overall.